site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -190,6 +190,14 @@ The JSON payload that is sent is a `FleetAutoscaleReview` data structure and a `
 The `FleetAutoscaleResponse`'s `Replica` field is used to set the target `Fleet` count with each sync interval, thereby
 providing the autoscaling functionality.
 
+{{% feature publishVersion="1.54.0" %}}
+{{< beta title="Fleet Metadata in Webhook Requests" gate="FleetAutoscaleRequestMetaData" >}}
+
+When the `FleetAutoscaleRequestMetaData` feature gate is enabled by default, the Fleet's `labels` and 
+`annotations` are included in the `FleetAutoscaleRequest`, allowing webhooks to make scaling decisions based on 
+Fleet-specific metadata.
+{{% /feature %}}
```

Comment:
This would bring things up to date for what is current, and also move it to beta with the next release 👍🏻 

```suggestion
{{% feature expireyVersion="1.54.0" %}}
{{< alpha title="Fleet Metadata in Webhook Requests" gate="FleetAutoscaleRequestMetaData" >}}
{{% /feature %}}

{{% feature publishVersion="1.54.0" %}}
{{< beta title="Fleet Metadata in Webhook Requests" gate="FleetAutoscaleRequestMetaData" >}}
{{% /feature %}}

When the `FleetAutoscaleRequestMetaData` feature gate is enabled, the Fleet's `labels` and 
`annotations` are included in the `FleetAutoscaleRequest`, allowing webhooks to make scaling decisions based on 
Fleet-specific metadata.
```

Reply:
Thank you! I've made the changes.
https://github.com/googleforgames/agones/pull/4365/commits/dd07e2627b1efe4a51302a0f54331cee64199371
https://github.com/googleforgames/agones/pull/4365/commits/2a9e4ef774c4e166a752d78dbba8740a4f9fe9a7